include/megaapi.h

```diff
@@ -8092,6 +8092,25 @@ class MegaApi
          */
         void getPaymentId(MegaHandle productHandle, MegaRequestListener *listener = NULL);
 
+        /**
+         * @brief Get the payment URL for an upgrade
+         *
+         * The associated request type with this request is MegaRequest::TYPE_GET_PAYMENT_ID
+         * Valid data in the MegaRequest object received on callbacks:
+         * - MegaRequest::getNodeHandle - Returns the handle of the product
+         *
+         * Valid data in the MegaRequest object received in onRequestFinish when the error code
+         * is MegaError::API_OK:
+         * - MegaRequest::getLink - Payment ID
+         *
+         * @param productHandle Handle of the product (see MegaApi::getPricing)
+         * @param lastPublicHandle Last public handle accessed by the user in the last 24h
```

Comment:
Perhaps specify that we are talking about public handles of nodes

Reply:
Fixed!